src/components/form/fields/schema-field.ts

```diff
@@ -150,7 +150,9 @@ export class SchemaField extends React.Component<FieldProps> {
         const { errorSchema, schema } = this.props;
 
         if (!this.isInvalid()) {
-            return schema.description;
+            const helperText = schema.lime?.component?.props?.helperText;
+
+            return schema.description || helperText;
```

Comment:
If `helperText` has been explicitly set, shouldn't we prefer showing that over `schema.description` if both exist?

```suggestion
            return helperText || schema.description;
```

Reply:
makes sense